src/app/main.rs

```diff
@@ -363,7 +368,9 @@ where
                 //    is retryable.
                 let dst_route_layer = phantom_data::layer()
                     .push(insert_target::layer())
-                    .push(metrics::layer::<_, classify::Response>(retry_http_metrics.clone()))
+                    .push(metrics::layer::<_, classify::Response>(
+                        retry_http_metrics.clone(),
```

Comment:
looks like there were a lot of unrelated formatting changes here...not super important, though.

Reply:
this file is now in the hands of rustfmt